domains/client/domain-executor/src/core_bundle_processor.rs

```diff
@@ -63,6 +65,7 @@ impl<Block, SBlock, PBlock, Client, SClient, PClient, Backend, E>
 where
     Block: BlockT,
     SBlock: BlockT,
+    Block::Extrinsic: Into<SBlock::Extrinsic>,
```

Comment:
We use `crate::utils::translate_number_type()` and `crate::utils::translate_block_hash_type()` before, perhaps we should also use something similar to the extrinsic so that they are at least consistent.

Reply:
I'm not very strict about this because the translate_* functions are essentially encoding and decoding again. I felt direct trait bound are easier to read comparatively. Any specific reason why translate then then we already know the hard types behind the trait matches anyway

Reply:
Yeah, they are technically the same at the moment, and all `Block::Extrinsic` are essentially the type `sp_runtime::OpaqueExtrinsic`. I don't remember who initially used the translate way, if it was me, I guess it's likely I don't want the `Block` generics to become more complex (it's somehow more comfortable to have one `BlockT` on `Block`).

Actually, no strong preference on which approach, but I'd like us not to use both of them at the same time, could you note a TODO somewhere to unify this usage?